src/apps/spoc_verifier.py

```diff
@@ -11,44 +12,41 @@
 
 main_col, geo_col, action_col = st.beta_columns([2, 1, 0.5])
 
-species = pd.DataFrame(
-    [
-        {
-            "Paper ID": "A",
-            "Instance ID": "A02",
-            "Species - Genus": None,
-            "Time": None,
-            "Place": None,
-        },
-        {
-            "Paper ID": "A",
-            "Instance ID": "A03",
-            "Species - Genus": "Tonicella lineata",
-            "Time": "1982",
-            "Place": "Grimes Point",
-        },
-        {
-            "Paper ID": "B",
-            "Instance ID": "B01",
-            "Species - Genus": None,
-            "Time": None,
-            "Place": None,
-        },
-    ]
-)
+species = pd.read_json("data/species-records.json")
+
+grid_options = GridOptionsBuilder.from_dataframe(species)
+
+# Grid options
+grid_options.configure_selection("single", use_checkbox=True)
+grid_options.configure_pagination(paginationAutoPageSize=True)
 
 with main_col:
     """
     # SPOC Verifier
     """
-    st.dataframe(species)
+    records_grid = AgGrid(
+        species,
+        gridOptions=grid_options.build(),
+        data_return_mode=DataReturnMode.FILTERED,
+        update_mode=GridUpdateMode.MODEL_CHANGED,
+        allow_unsafe_jscode=True,
```

Comment:
😳Unsafe JS code?

Reply:
Good catch! This is needed if we inject JS code (which we aren't doing now). Removing.

Reply:
I verified that it defaults to False.